pages/settings/wallets/RenameWallet.tsx

```diff
@@ -16,11 +17,7 @@ export function RenameWallet() {
     wallets[selectedWalletId].name || ""
   );
   return (
-    <TouchableWithoutFeedback
-      onPress={() => {
-        Keyboard.dismiss();
-      }}
-    >
+    <DismissableKeyboardView>
```

Comment:
I realized we add this on every page, can it go in `_layout.tsx`?

Reply:
Not exactly every page, also not sure if we should add it because we clubbed it with `TouchableWithoutFeedback` and it does Keyboard.dismiss() on press

Reply:
ah ok, thanks!